tests/tests_unit/test_testing.py

```diff
@@ -12,7 +13,7 @@
 def test_ensure_all_apis_are_available_on_cognite_mock():
     mocked_apis = all_mock_children(CogniteClientMock())
     available = {v.__class__ for v in mocked_apis.values()}
-    expected = set(all_subclasses(APIClient))
+    expected = set(all_subclasses(APIClient)) - {BetaWorkflowAPIClient}
```

Comment:
Why this exception? Why is it difficult to add to the mock client?

Reply:
I followed what another PR had been doing. Thought it was the way to introduce beta methods.
I'll update.